widgets/linechart/linechart.go

```diff
@@ -519,3 +531,22 @@ func (lc *LineChart) maxXValue() int {
 	}
 	return maxLen - 1
 }
+
+const (
```

Comment:
(optional) Since these consts are only used within the minMax function below, we could as well define them inside the function. Encapsulating them in might give a better explanation of what they are for.

Or we could as well not use the consts and just go with literal zeroes, up to you.